integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -95,6 +136,28 @@ public static void initialize(String appPropsFile) throws Exception {
               + "\n"
               + clnResult.stderr());
     }
+
+    if (System.getenv("JENKINS") != null) {
+      logger.info("Creating " + resultRoot + "/acceptance_test_tmp");
+      TestUtils.exec(
+          "/usr/local/packages/aime/ias/run_as_root \"mkdir -p "
+              + resultRoot
+              + "/acceptance_test_tmp\"");
+      TestUtils.exec(
+          "/usr/local/packages/aime/ias/run_as_root \"chmod 777 "
+              + resultRoot
+              + "/acceptance_test_tmp\"");
+      logger.info("Creating " + pvRoot + "/acceptance_test_pv");
+      TestUtils.exec(
+          "/usr/local/packages/aime/ias/run_as_root \"mkdir -p "
+              + pvRoot
+              + "/acceptance_test_pv\"");
+      TestUtils.exec(
+          "/usr/local/packages/aime/ias/run_as_root \"chmod 777 "
+              + pvRoot
+              + "/acceptance_test_pv\"");
+    }
+
```

Comment:
Just an FYI - We'll need a different approach for setting up directories on the new public Jenkins boxes.

Reply:
@bsmoyers has already created the new cluster on OCI and created the OCI file system.  This will be available through NFS PV's, but won't be auto-mounted on any of the Wercker nodes.  So, as Tom points out, we'll have to copy files with exec or cp onto a pod that has the proper volume.

Reply:
Sure. we will work out a new approach at that time.